yaksh/templates/yaksh/quit.html

```diff
@@ -1,11 +1,15 @@
 {% extends "base.html" %}
 
-{% block pagetitle %} 
-<div class="yakshnav fixed-top" style=" padding-top: 0.5%;padding-bottom: 0.5%;">
-    <a class="navbar-brand" href="{{ URL_ROOT }}/exam/"  >
-    <img src="{{ URL_ROOT }}/static/yaksh/images/yaksh_banner.png" alt="YAKSH">
-    </img>
-    </a> 
+{% block nav %} 
+<div class="container-fluid yakshnav">
+  <nav class="navbar fixed-top navbar-expand-lg yakshheading yakshnav">
+    <div class="container">
+      <a class="navbar-brand">
+          <img src="{{ URL_ROOT }}/static/yaksh/images/yaksh_banner.png" alt="YAKSH" style="margin-top: -3px; margin-left:-15px">
```

Comment:
This is exactly same as above, so CSS can be in separate file and common to both instead of inline CSS.

Reply:
Done, removed